csvkit.md

```diff
@@ -0,0 +1,210 @@
+
+Pipeability is a core feature of csvkit. Of course, you can always write your output to a file using `>`, but many times it makes more sense to use pipes for speed and brevity.
+
+Of course, we can also pipe `in2csv`, combining all our previous operations into one:
+
+```bash
+in2csv ne_1033_data.xlsx | csvcut -c county,item_name,quantity | csvlook | head
+```
+
+
+## Examining the data
+
+### csvstat: statistics without code
+
+In the previous section we saw how we could use `csvlook` and `csvcut` to peek at slices of our data. This is a good starting place for diving into a dataset, but in practice we usually want to get the widest possible view before we start diving into specifics.
+
+`csvstat` is designed to give us just such a broad picture of our data. It is inspired by the summary() function from the computational statistics programming language [R](http://www.r-project.org/).
+
+Let's examine summary statistics for some selected columns from our data (remember you can use `csvcut -n data.csv` to see the columns in the data):
+
+```bash
+csvcut -c county,acquisition_cost,ship_date data.csv | csvstat
+```
+
+`csvstat` algorithmically infers the type of each column in the data and then performs basic statistics on it. The particular statistics computed depend on the type of the column.
+
+In this example the first column, `county` was identified as type "unicode" (text). We see that there are `35` counties represented in the dataset and that `DOUGLAS` is far and away the most frequently occurring. A quick Google search shows that there are `93` counties in Nebraska, so we know that either not every county received equipment or that the data is incomplete. We can also find out that Douglas county contains Omaha, the state's largest city by far.
+
+The `acquisition_cost` column is type "float" (number including a decimal). We see that the largest individual cost was `412,000`. (Probably dollars, but let's not presume.) Total acquisition costs were `5,438,254`.
+
+Lastly, the `ship_date` column shows us that the earliest data is from ``1984`` and the latest from `2054`. From this we know that there is invalid data for at least one value, since presumably the equipment being shipped does not include time travel devices. We may also note that an unusually large amount of equipment was shipped in April, 2013.
+
+As a journalist, this quick glance at the data gave me a tremendous amount of information about the dataset. Although we have to be careful about assuming to much from this quick glance (always double-check the numbers!) it can be an invaluable way to familiarize yourself with a new dataset.
```

Comment:
```suggestion
As a journalist, this quick glance at the data gave me a tremendous amount of information about the dataset. Although we have to be careful about assuming too much from this quick glance (always double-check the numbers!) it can be an invaluable way to familiarize yourself with a new dataset.
```